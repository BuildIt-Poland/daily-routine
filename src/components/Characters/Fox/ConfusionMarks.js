import styled from 'styled-components';

import ConfusionMarksBase from '../ConfusionMarks';

const ConfusionMarks = styled(ConfusionMarksBase)`
  top: 12.6rem;
  left: 6rem;
`;

export default ConfusionMarks;
