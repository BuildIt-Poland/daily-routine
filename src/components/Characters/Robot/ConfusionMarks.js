import styled from 'styled-components';

import ConfusionMarksBase from '../ConfusionMarks';

const ConfusionMarks = styled(ConfusionMarksBase)`
  top: -3.6rem;
  left: 6rem;
`;

export default ConfusionMarks;
