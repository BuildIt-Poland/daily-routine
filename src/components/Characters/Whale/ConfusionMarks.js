import styled from 'styled-components';

import ConfusionMarksBase from '../CharactersDecors/ConfusionMarks';

const ConfusionMarks = styled(ConfusionMarksBase)`
  top: 9.8rem;
  left: 5.2rem;
`;

export default ConfusionMarks;
