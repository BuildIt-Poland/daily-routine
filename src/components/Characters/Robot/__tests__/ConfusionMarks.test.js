import React from 'react';
import { create } from 'react-test-renderer';

import ConfusionMarks from '../ConfusionMarks';

jest.mock('../../ConfusionMarks', () => 'ConfusionMarks');

describe('COMPONENT - Characters Robot GenderFlower', () => {
  it('renders correctly', () => {
    const component = create(<ConfusionMarks />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
