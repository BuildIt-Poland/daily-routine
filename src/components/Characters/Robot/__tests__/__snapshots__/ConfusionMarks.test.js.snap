// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Characters Robot GenderFlower renders correctly 1`] = `
.c0 {
  top: -3.6rem;
  left: 6rem;
}

<ConfusionMarks
  className="c0"
/>
`;
